gradoop-flink/src/main/java/org/gradoop/flink/model/api/epgm/BaseGraphCollectionOperators.java

```diff
@@ -79,6 +303,36 @@ default GC verifyGraphsContainment() {
    */
   GC callForCollection(UnaryBaseGraphCollectionToBaseGraphCollectionOperator<GC> operator);
 
+  /**
+   * Calls the given binary collection to value operator using that
+   * graph collection and the input graph collection.
+   *
+   * @param operator        binary collection to value operator
+   * @param otherCollection second input collection for operator
+   * @param <T> return type
+   * @return result of given operator
+   */
+  <T> T callForCollection(BinaryBaseGraphCollectionToValueOperator<GC, T> operator, GC otherCollection);
+
+  /**
+   * Calls the given binary collection to collection operator using that
+   * graph collection and the input graph collection.
```

Comment:
You could move some words one line up, although this does not fit on one line anyways, so it should be okay.